File: apps/s-2-admin/src/api/impersonation/ImpersonationWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { DateTimeFilter } from "../../util/DateTimeFilter";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";

export type ImpersonationWhereInput = {
  id?: IntFilter;
  createdAt?: DateTimeFilter;
  impersonatedUser?: UserWhereUniqueInput;
  impersonatedBy?: UserWhereUniqueInput;
};

File: apps/cactus-demo-dotnet/src/APIs/Impersonation/Dtos/ImpersonationWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class ImpersonationWhereInput
{
    public int? Id { get; set; }

    public DateTime? CreatedAt { get; set; }

    public int? ImpersonatedUser { get; set; }

    public int? ImpersonatedBy { get; set; }
}
